packages/liquid-html-parser/src/stage-2-ast.ts

```diff
@@ -1285,14 +1289,11 @@ function buildAst(
           name: node.name,
           position: position(node),
           source: node.source,
-          paramName: {
-            type: NodeTypes.TextNode,
-            value: node.paramName.value,
-            position: position(node.paramName),
-            source: node.paramName.source,
-          },
+          paramName: toTextNode(node.paramName.paramNameContent),
```

Comment:
`@param {String} [param_name=123]`

This allows us to refer to the `node.paramName`, which will be the node referring to `param_name` with the associated location info